charts/datadog/values.yaml

```
@@ -830,6 +835,9 @@ agents:
   #
   enabled: true
 
+  # agents.shareProcessNamespace -- Set the process namespace sharing on the Datadog Daemonset
+  shareProcessNamespace: true
```

same comment about default value to false 

```suggestion
  shareProcessNamespace: false
```